From 103aa15b1cfd469b0ed5cdd7a8f478f16bfa9b89 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 11 Jan 2016 14:16:00 +0800
Subject: [PATCH] MLK-11987 cpufreq: imx: remove unused value assignment

Remove the unused value assignment reported by
coverity check(CID 17279).

Signed-off-by: Bai Ping <ping.bai@nxp.com>
(cherry picked from commit 3c4a0d737e615ac27c04e2612036c44925fe8757)
---
 drivers/cpufreq/imx6q-cpufreq.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/cpufreq/imx6q-cpufreq.c b/drivers/cpufreq/imx6q-cpufreq.c
index a9fd553..c57e08e 100644
--- a/drivers/cpufreq/imx6q-cpufreq.c
+++ b/drivers/cpufreq/imx6q-cpufreq.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013-2015 Freescale Semiconductor, Inc.
+ * Copyright (C) 2013-2016 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -138,7 +138,7 @@ static int imx6q_set_target(struct cpufreq_policy *policy, unsigned int index)
 		if (freq_hz > clk_get_rate(pll2_pfd2_396m_clk))
 			clk_set_parent(secondary_sel, pll2_bus);
 		else
-			ret = clk_set_parent(secondary_sel, pll2_pfd2_396m_clk);
+			clk_set_parent(secondary_sel, pll2_pfd2_396m_clk);
 		clk_set_parent(step_clk, secondary_sel);
 		clk_set_parent(pll1_sw_clk, step_clk);
 	} else {
-- 
1.8.0

